site/_data/banner.yml

```
@@ -0,0 +1,6 @@
+type: info
+text: 'PWA Summit 22 is ready for you, with talks from the community and organizers along with workshops and more.'
+actions:
+  - text: Get your ticket today
+    href: https://pwasummit.com/register
```

Should this be .org? As the other site doesn't work.

```suggestion
    href: https://pwasummit.org/register
```